machine/machine_core/machine.py

```diff
@@ -187,7 +187,9 @@ def journal_messages(self, syslog_ids, log_level, cursor=None):
             cursor_arg = ""
 
         cmd = "journalctl 2>&1 %s -o cat -p %d %s || true" % (cursor_arg, log_level, matches)
+        print("J", cmd)
         messages = self.execute(cmd).splitlines()
+        print("M", messages)
```

Comment:
Debugging leftover? If not, this shoudl be a bit more verbose.

Reply:
Yes, oops.

Reply:
Fixed.